meson.build

```diff
@@ -269,6 +269,12 @@ if get_option('plugin_thunderbolt')
   conf.set('HAVE_THUNDERBOLT', '1')
 endif
 
+if get_option('plugin_flashrom')
+  flashrom = find_program('flashrom', required: false)
+  libflashrom = cc.find_library('libflashrom', required: false)
+  conf.set('HAVE_FLASHROM', '1')
```

Comment:
I do not quite get it how it works. Both flashrom and libflashrom are not required?
And we set config that we have flashrom?

Reply:
my approach is that not everyone wants to use libflashrom/flashrom plugin. `meson` will check if the system has installed above dependencies and set appropriate info during build process